.github/workflows/ci.yml

```diff
@@ -48,17 +48,18 @@ jobs:
       - name: Setup Vertica
         timeout-minutes: 15
         run: |          
-          docker pull vertica/vertica-ce:12.0.0-0
           docker run -d -p 5433:5433 -p 5444:5444 \
             --mount type=volume,source=vertica-data,target=/data \
             --name vertica_ce \
-            vertica/vertica-ce
+            vertica/vertica-ce:12.0.1-0
```

Comment:
When we update the server version after each GA period, is this the only line that we will need updating now?

Reply:
correct